File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--creating-your-first-vsto-add-in-for-project_1.cs
        void Application_NewProject(Microsoft.Office.Interop.MSProject.Project pj)
        {
            MSProject.Task newTask = pj.Tasks.Add
                ("This text was added by using code", missing);
            newTask.Start = DateTime.Now;
            newTask.Duration = "3";
            newTask.ResourceNames = "Rob Caron, Kelly Krout";
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--creating-your-first-vsto-add-in-for-project_1.vb
    Private Sub Application_NewProject(ByVal pj As Microsoft.Office.Interop.MSProject.Project) Handles Application.NewProject
        Dim newTask As MSProject.Task
        newTask = pj.Tasks.Add _
        ("This text was added by using code")
        newTask.Start = DateTime.Now
        newTask.Duration = "3"
        newTask.ResourceNames = "Rob Caron, Kelly Krout"
    End Sub